tests/integration_tests/vectorstores/test_azuresearch.py

```diff
@@ -0,0 +1,93 @@
+import os
+import time
+
+import openai
+import pytest
+from dotenv import load_dotenv
+
+from langchain.embeddings.openai import OpenAIEmbeddings
+from langchain.vectorstores.azuresearch import AzureSearch
+
+load_dotenv()
+
+# Azure OpenAI settings
+openai.api_type = "azure"
+openai.api_base = os.getenv("OPENAI_API_BASE", "")
+openai.api_version = "2023-03-15-preview"
```

Comment:
How about using GA'd version ""2023-05-15" instead of the preview "2023-03-15-preview"?

Reply:
Yep, this was done before GA.